ecchronos-binary/src/pylib/ecchronoslib/rest.py

```diff
@@ -77,7 +79,15 @@ def request(self, url, method='GET'):
         try:
             request = Request(request_url)
             request.get_method = lambda: method
-            response = urlopen(request)
+            cert_file = os.getenv("ECCTOOL_CERT_FILE")
+            key_file = os.getenv("ECCTOOL_KEY_FILE")
+            ca_file = os.getenv("ECCTOOL_CA_FILE")
+            if cert_file and key_file and ca_file:
+                context = ssl.create_default_context(cafile=ca_file)
+                context.load_cert_chain(cert_file, key_file)
+                response = urlopen(request, context=context)
```

Comment:
Will the request not be done with http by default? Wouldn't this mean that the user will always have to specify url if he uses certificates? This does not seem right.